src/wp-admin/includes/plugin-install.php

```diff
@@ -399,6 +399,7 @@ function display_plugins_table() {
 			);
 			break;
 		case 'install_plugins_featured':
+			echo '<br>';
```

Comment:
🤩 I just want to take a moment and thank/congratulate you for using the normative well-formed markup syntax instead of propagating the benign-but-misunderstood-erroneous role of the self-closing flag.

you have made the world a little bit more correct.